### ISSUE TITLE
Feature Request: add a custom LargeIcon for notification

### ISSUE THREAD

**Issue by Paolo7297** [gotify/android#256]
It would be nice to have another field inside `client::notification`, maybe called `imageUrl` to set a custom LargeIcon (the one inline with the text in the notification).

**Comment by jmattheis** [comment]
I don't know if this is really needed, as you can already add custom image via Markdown. I'd be open to wait for feedback from other users.

**Comment by Paolo7297** [comment]
Sorry for the stupid question, but how can I replace the LargeIcon (so the one which is inline with the text) with markdown? I already tried to add image in markdown , but it's only available on web ui, on android is rendered as a small block with "obj" text inside.

I know there's `bigImageUrl`, but the image covers half of the screen, instead having inline with the text would be much more clear.

**Comment by jmattheis** [comment]
What do you mean with large icon, the image left to the text of the message content? What's the markdown image url that doesn't work on the android app but on the webui?

**Comment by Paolo7297** [comment]
Yes, this one. You set in the notification using the `setLargeIcon` method.

I just tried with https://placekitten.com/400/300 with the same result as images in my webserver.

**Comment by jmattheis** [comment]
You can change the image globally for all messages from an application, but for a single message you cannot override it. You can embed the image with markdown into the message like this (using placekitten):

![image](https://user-images.githubusercontent.com/14895212/204618904-ed6b4aef-cf4a-4a3a-beea-1c1597683fe2.png)

**Comment by Paolo7297** [comment]
Yes, that's why I'm asking for a feature to override it.

**Comment by jmattheis** [comment]
Yes, I understand this, but as said in my first comment, I think it not a feature that is needed by many users, so let's wait for comments from other users.
